services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeController.kt

```diff
@@ -93,7 +146,27 @@ class HankeController(
     }
 
     /** Add one hanke. This method will be called when we do not have id for hanke yet */
-    @PostMapping
+    @PostMapping(consumes = [APPLICATION_JSON_VALUE], produces = [APPLICATION_JSON_VALUE])
+    @Operation(
+        summary = "Create new hanke",
+        description =
+            "Create a new hanke. A valid new hanke must comply with the restrictions defined in HankeValidator."
```

Comment:
This should also describe what happens when a new hanke is created. So we don't have to document that separately someplace else.

Things like:
- A unique Hanketunnus is created for the new hanke and returned in the response
- The status of the created hanke is set automatically:
  - PUBLIC if all mandatory fields are filled, i.e. it will be visible to everyone.
  - DRAFT if the fields haven't been set

And other things the caller of this API might be interested in.

Reply:
Added those you mentioned. Hanke has Schema definition annotations now and that should take care of the details. But these points are good to have in the endpoint description also.